ckanfunctionaltests/api/conftest.py

```diff
@@ -252,9 +254,9 @@ def stable_org_with_datasets(inc_fixed_data):
 
 
 @pytest.fixture()
-def stable_dataset(inc_fixed_data):
+def stable_dataset(variables, inc_fixed_data):
     return set_ckan_vars(
         get_example_response(
-        "stable/search_dataset.inner.test.json"
+        "stable/search_dataset{}.inner.test.json".format('-2.9' if variables['ckan_version'] == '2.9' else '')
```

Comment:
Thanks for addressing my earlier review comments 👍 If it were me, I'd still be pretty uneasy about the introduction of these hardcoded `2.9` conditionals here...

Reply:
At the moment the output from 2.9 is quite different from 2.7 and 2.8, once 2.9 becomes more stable and established I think the code will have to be re-reviewed and the test updated accordingly but I don't want to over engineer it in case 3.0 is also very different.